packages/block-library/src/query-total/block.json

```
@@ -0,0 +1,45 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "core/query-total",
+	"title": "Query Total",
+	"category": "theme",
+	"parent": [ "core/query-pagination", "core/query" ],
```

I would suggest we change this to just have the query as it's ancestor. That way it can appear anywhere deeply nested in the query. 

```suggestion
	"ancestor": [ "core/query" ],
```

---

Good call 👍